### ISSUE TITLE
Patch to resolve the error "Makefile:109: recipe for target 'build/ColumnListView.o' failed"

### ISSUE THREAD

**Issue by Giova84** [HaikuArchives/Helios#6]
I made a diff file to resolve the error "Makefile:109: recipe for target 'build/ColumnListView.o' failed" while attempt to build Helios.

I guess that I should make a pull request, but I have no idea how.

However there is my diff file: once applied this simple change, Helios will be able to compile fine:
http://softwaredepot.altervista.org/docs/Helios_patch.diff

**Comment by scottmc** [comment]
See if this helps:
https://help.github.com/articles/creating-a-pull-request/
Or perhaps others know of a better set of instructions for creating pull
requests for HaikuArchives?

**Comment by Giova84** [comment]
Hi Scott,

Well, I also read a more complete how-to about pull requests, but unfortunately is not very clear to me (is not a language issue); however I am working on some more improvements for Helios: I will keep the diffs files for when I will be able to submit pull requests, otherwise I hope that someone know of a better set of instructions for creating pull requests for HaikuArchives.

I'm sorry for the noise.
Kind regards.

**Comment by pulkomandy** [comment]
If you work from Haiku there are some tools to make it a little simpler (but still a bit technical, unfortunately).

```
pkgman install hub
```

I assume you have cloned the git repository and made your changes there (so you can see them with 'git diff', for example).

   git commit

This will create a "commit" with all your changes. It will ask you to enter a description for them (don't worry, we can edit it later on if it isn't appropriate).

You can create multiple separate commits for different (or related) changes this way.

When you are ready, use the hub tool to create the pull-request:

```
hub fork
```

This will create your own "fork" of the project on github at https://github.com/giova84/helios

```
git push giova84
```

This will put all your new commits into said "fork"

```
hub pull-request
```

And this will automatically create the pull request from that fork.

Once this is done, if there are further changes to make, you only need to repeat git commit, and git push giova84. The pull request is automatically updated, as long as it is open.

If this is still too complex, no worries, we can also merge the changes manually. However, in that case you will not get complete credit in the git commits if you provide simple "diff" files. If you can live with that, submitting diffs is ok.

**Comment by waddlesplash** [comment]
Looks like this was fixed in b870e477789928027dd4f3a444219cb820944678.
